src/OpenLoco/Map/WaveManager.cpp

```diff
@@ -33,16 +22,16 @@ namespace OpenLoco::Map::WaveManager
         Pos2(0, -32),
     };
 
-    static LocoFixedVector<Wave> waves()
+    static FixedVector<Wave, 64> waves()
     {
-        return LocoFixedVector<Wave>(_waves);
+        return FixedVector(getGameState().waves);
     }
 
     // 0x0046956E
     void createWave(SurfaceElement& surface, const Map::Pos2& pos)
     {
         const auto waveIndex = getWaveIndex(pos);
-        if (!_waves[getWaveIndex(pos)].empty())
+        if (!getGameState().waves[getWaveIndex(pos)].empty())
```

Comment:
I think it might look better to introduce a `rawWaves()` function for the wave manager. Will then follow that style for other managers exposing the fixed vector and internally having a raw version that just gets the game state array.